### PR TITLE
[TEST] Add integration tests to create single comment

### LINKED ISSUE #331: [TEST] Single comment create - POST /v1/comments
### Is your feature request related to a problem? Please describe.

Update tests for `createSingleComment.js` controller

### Describe the solution you'd like

Test should check that the following are covered:

- test 200/201 success
- test to `POST` with missing required parameters
- test to `POST` with empty body
- test 401 authentication error
- test 404 not found error

### Additional context

### PR DESCRIPTION
**Pull request by piouson**
**Before submitting your PR for review**

- [x] Run `npm run lint` to find errors in code syntax/format
- [x] Run `npm run lint:fix` to fix all auto-fixable errors in source code and auto-format with prettier
- [x] Ensure you fix any linting errors displayed after running any of the above commands

**What does this PR do?**

Fixes #331 

**description of Task to be completed?**

- [x] add test for status 201 success
- [x] add test for status 422 missing required parameters
  - missing `content`
  - missing `ownerId`
- [x] add test for status 422 missing authorization token

**How should this be manually tested?**

```bash
npm install
npm run test
```

**Any background context you want to provide?**

**What is the link to the issue on Github?**

Issue #331 

**Questions:**

### CODE REVIEW

File: __test__/api/comments/createSingleComment.test.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+const app = require("../../../server");
+const supertest = require("supertest");
+const CommentModel = require("../../../models/comments");
+const request = supertest(app);
+
+describe("POST /comments", () => {
+  let comment;
+
```
Comment (simply-alliv) [inline __test__/api/comments/createSingleComment.test.js:8]: Hey, where is the beforeEach function where you create the comment that you delete in the afterEach function.

File: __test__/api/comments/createSingleComment.test.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  afterEach(async () => {
+    await CommentModel.findByIdAndDelete(comment.commentId);
+    comment = null;
+  });
+
+  test("Should create comment", async () => {
```
Comment (simply-alliv) [inline __test__/api/comments/createSingleComment.test.js:14]: I'd also suggest you use the toEqual() matcher the way I used it in the getAllReplies test. Check it out and tell me what you think. It's just a suggestion though. This works fine too. It's just verbose.

File: __test__/api/comments/createSingleComment.test.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        content: "this is a comment",
+      });
+    comment = res.body;
+    expect(res.status).toBe(422);
+    expect(res.body.status).toEqual("error");
+    expect(res.body.error).toBeTruthy();
+    //expect(res.body.message).toBeTruthy(); // or {}
+  });
+
+  test("Should return 401 error when 'Authorization' token missing", async () => {
```
Comment (simply-alliv) [inline __test__/api/comments/createSingleComment.test.js:67]: The 401 is not for a missing authorization token. A missing authorization token would return a 422 response due to the validation middleware. In this case, the token is not missing but it is unauthorized.

Comment (piouson) [reply __test__/api/comments/createSingleComment.test.js]: should be sorted now, have another look please..

File: __test__/api/comments/createSingleComment.test.js (hunk 1 above)
Comment (piouson) [reply __test__/api/comments/createSingleComment.test.js:8]: since the endpoint creates the entry in db, only `afterEach` was required to cleanup db

**Pull request merged**